Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCiscoASA.yaml

```diff
@@ -284,6 +284,13 @@ ParserQuery: |
                 | project-away Activity, AdditionalExtensions, ApplicationProtocol, Device*, Destination*, EndTime, ExternalID, File*, Flex*, IndicatorThreatType, Malicious*, Old*, Process*, ReceiptTime, ReceivedBytes, Remote*, Report*, Request*, Source*, SentBytes, StartTime, TenantId, Threat*;
       let unparsedData = allLogs
                 | where isempty(SourceIP)
+                | where Message has tostring(dstportnumber)
+                        and ((array_length(src_or_any) == 0 or Message has_any(src_or_any)) 
```

Comment:
Fixed the bug for CiscoASA and would verify for Checkpoint as well.
Kept source and destination separate only because we need it for setting "ASimMatchingIpAddr" parameter.